simulation_parameters/Constants.cs

```diff
@@ -24,7 +24,15 @@ public static class Constants
     /// </summary>
     public const float SPAWN_SECTOR_SIZE = 120.0f;
 
-    public const int CLOUD_SPAWN_SCALE_FACTOR = 10000;
+    /// <summary>
+    ///   Scale factor for density of compound cloud spawns
+    /// </summary>
+    public const int CLOUD_SPAWN_DENSITY_SCALE_FACTOR = 10000;
+
+    /// <summary>
+    ///   Scale factor for amount of compound in each spawned cloud
+    /// </summary>
+    public const float CLOUD_SPAWN_AMOUNT_SCALE_FACTOR = 0.75f;
```

Comment:
I got the feeling that this variable is kind of useless? Because the cloud amounts are all specified in the JSON file, which you also modified in this PR.

Reply:
The cloud densities and amounts are used together to calculate two things: clouds themselves, and the GUI display for amount of compounds in each patch. Given that these aren't the same units, I thought it would be sensible to have an easily visible constant to translate between the two.

Reply:
Still I feel like the extra conversion factor is pretty unnecessary here. I think the display in the GUI might already have a multiplier there, so to keep the previous GUI value showing even with the different amounts actually spawning, that GUI display multiplier would be the one I think should be changed.

Reply:
I guess I can live with this change that adds one extra step to understanding what the values in biomes.json mean, so that we can focus on #3389

Reply:
Turns out there is no other constant for the GUI - it displays the raw `amount * density` value. So I definitely think keeping this constant makes sense.